.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug with the specification.
+
+---
+
+<!--
+Thank you for reporting a possible bug in ECMA-402.
+
+If this is a bug with a specific implementation of ECMA-402, such as
+a JavaScript engine or a browser, you might need to report it on their bug
+tracker as well.
```

Comment:
Awesome recommendation! Should we also link the different engines' issue trackers? Doesn't seem to be a lot of effort on our end, but it might save the reporting a little time...